openassessment/xblock/static/sass/oa/utilities/_developer.scss

```diff
@@ -1077,6 +1077,18 @@
         }
       }
     }
+
+    // TODO: Remove this in AU-1433
```

Comment:
Nit: Can you explain here the goal of the referenced ticket ie: "Decide feature is no longer new and remove tagging"